Resources/translations/SonataClassificationBundle.es.xliff

```diff
@@ -158,6 +158,10 @@
                 <source>classification.tree_mode</source>
                 <target>classification.tree_mode</target>
             </trans-unit>
+            <trans-unit id="no_tag_found">
+                <source>no_tag_found</source>
+                <target>No tag found.</target>
```

Comment:
If we don't have a translation, please use the source key for the target in this case:

```xml
<trans-unit id="no_tag_found">
    <source>no_tag_found</source>
    <target>no_tag_found</target>
</trans>
```